packages/pds/src/app-view/api/app/bsky/feed/getAuthorFeed.ts

```diff
@@ -64,8 +63,15 @@ export default function (server: Server, ctx: AppContext) {
               .whereRef('post_embed_image.postUri', '=', 'feed_item.postUri'),
           )
       } else if (params.filter === 'posts_no_replies') {
-        // only posts, no replies
-        feedItemsQb = feedItemsQb.where('post.replyParent', 'is', null)
+        feedItemsQb = feedItemsQb
+          // only posts, no replies
+          .where('post.replyParent', 'is', null)
+          // or any reposted replies
+          .orWhere((qb) => {
+            return qb
+              .where('originatorDid', '=', actorDid)
```

Comment:
i actually don't think this line is necessary, because we're already only selecting items originating from the actor